### PR TITLE
chore: Update online store docstrings

### PR DESCRIPTION
**Pull request by felixwang9817**
Signed-off-by: Felix Wang <wangfelix98@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/feast/infra/online_stores/online_store.py [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
 class OnlineStore(ABC):
     """
-    OnlineStore is an object used for all interaction between Feast and the service used for online storage of
-    features.
+    An online store defines the interface that Feast uses to interact with the the storage system
+    that handles online features.
```
Comment (achals) [inline sdk/python/feast/infra/online_stores/online_store.py:31]:
```suggestion
    The interface that Feast uses to interact with the the storage system
    that handles online features.
```

**Review by achals — APPROVED** [review]

File: sdk/python/feast/infra/online_stores/online_store.py (hunk 1 above)
Comment (felixwang9817) [reply sdk/python/feast/infra/online_stores/online_store.py]: fixed

Comment (kevjumba) [reply sdk/python/feast/infra/online_stores/online_store.py]: also you have duplicate the

File: sdk/python/feast/infra/online_stores/datastore.py [hunk 2]
```diff
@@ -75,8 +75,10 @@ class DatastoreOnlineStoreConfig(FeastConfigBaseModel):
 
 class DatastoreOnlineStore(OnlineStore):
     """
-    OnlineStore is an object used for all interaction between Feast and the service used for offline storage of
-    features.
+    Google Cloud Datastore implementation of the online store interface.
```
Comment (adchia) [inline sdk/python/feast/infra/online_stores/datastore.py:78]: for this and redis, can you link to the corresponding reference doc on what the data model looks like? e.g. https://github.com/feast-dev/feast/blob/v0.23-branch/docs/specs/online_store_format.md#google-datastore-online-store-format

Comment (felixwang9817) [reply sdk/python/feast/infra/online_stores/datastore.py:78]: done

**Review by adchia — APPROVED** [review]
/lgtm

**Review by adchia — APPROVED** [review]
/lgtm

File: sdk/python/feast/infra/online_stores/sqlite.py [hunk 3]
```diff
@@ -50,8 +50,7 @@ class SqliteOnlineStoreConfig(FeastConfigBaseModel):
 
 class SqliteOnlineStore(OnlineStore):
     """
-    OnlineStore is an object used for all interaction between Feast and the service used for offline storage of
-    features.
+    SQLite implementation of the online store interface.
```
Comment (achals) [inline sdk/python/feast/infra/online_stores/sqlite.py:53]: Mention that this is not recommended in production?

**Pull request merged**